src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -4174,3 +4172,13 @@ def test_aks_create_with_apiserver_vnet_integration(self, resource_group, resour
         self.cmd(cmd, checks=[
             self.is_empty(),
         ])
+
+
+    @live_only()  # live only is required for test environment setup like `az login`
```

Comment:
Aha, before manifest release, I guess the case would not pass. May add the case name to `src/aks-preview/azcli_aks_live_test/configs/ext_matrix_default.json` to exclude it from check-in CI and cli runner (avoid IcM alert). Thanks!

Reply:
It can work now, tested locally

Reply:
Queued a [pipeline](https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=55555311&view=results) to test this case.

Reply:
we might add to skip now? the manifest didn't rollout to all regions. it works locally since we use regional ARM endpoint.

Reply:
oh, ok!